docs/tutorials/file-naming-convention.mdx

```diff
@@ -0,0 +1,149 @@
+
+### Text / Content
+`constitution.txt / *.constitution.txt` constitution in cleartext utf-8
+
+### Action
+`constitution.action / *.constitution.action` for action file content in json
+
+Example content:
+```json
+{
+    "type": "Governance proposal",
+    "description": "",
+    "cborHex": "841a000f4240581c1b7a61b...36a56ff490a4af6"
+}
+```
+
+### Vote
+`constitution.vote / *.constitution.vote` for the vote ballot in json
+
+Example content:
+```json
+{
+    "type": "Governance vote",
+    "description": "",
+    "cborHex": "8482582064e7cad9b3ece...1e235709ffa691001f6"
+}
+```
+
+
+## Delegate representatives (DReps) Files
+
+Below is a list of currently used files related to a Delegate Representative usage.
+
+### Signing Key
+Generated as a normal ed25519 key or derived from path `1852'/1815'/acc'/3/idx'` 
+`drep.skey / *.drep.skey` for the drep secret-key json (content like below)
+
+Example content:
+```json
+{
+    "type": "DRepSigningKey_ed25519",
+    "description": "Delegate Representative Signing Key",
+    "cborHex": "00000"
+}
+{
+    "type": "DRepExtendedSigningKey_ed25519_bip32",
+    "description": "Delegate Representative Signing Key",
+    "cborHex": "5840d84f8646463b03accf....da82c5dba3b1e4aa0df20b7"
+}
+```
+The keys derived from hardware wallets will have the keyword 'Hardware' in the description field like:
+`Hardware Delegate Representative Signing Key`
+
+### Verification Key
+`drep.vkey / *.drep.vkey` for the drep verification-key json (content like below)
+
+Example content:
+```json
+{
+    "type": "DRepVerificationKey_ed25519",
+    "description": "Delegate Representative Verification Key",
+    "cborHex": "00000"
+}
+{
+    "type": "DRepExtendedVerificationKey_ed25519_bip32",
+    "description": "Delegate Representative Verification Key",
+    "cborHex": "5820fd6b3cda64cf11f9b91ea3c88dffed906bc872f1e24566d68637c718788638d9"
+}
+```
+The keys derived from hardware wallets will have the keyword 'Hardware' in the description field like:
+`Hardware Delegate Representative Verification Key`
+
+### Identification File (ID)
+`drep.id / *.drep.id` for the drep id in bech format `drep_id1...`
+
+Example content:
+`drep_id12dggdndq3hhjzszukw5k5sulsesjux780s39h087s0p9vk6ylra`
+
+### Registration Certificate
+`drep-reg.cert / *.drep-reg.cert` for the registration certificate json
+
+Example content:
+```json
+{
+    "type": "CertificateConway",
+    "description": "DRep Registration Certificate",
+    "cborHex": "00000000"
+}
+```
+
+### Retirement Certificate
+`drep-ret.cert / *.drep-ret.cert` for the de-registration/retirement certificate json
+
+Example content:
+```json
+{
+    "type": "CertificateConway",
+    "description": "DRep Retirement Certificate",
+    "cborHex": "00000000"
+}
+```
+
+### Delegation Certificate
+`drep-deleg.cert / *.drep-deleg.cert` for the delegation certificate json
+
+Example content:
+```json
+{ 
+	"type": "CertificateShelley",
+	"description": "DRep Delegation Certificate",
```

Comment:
```suggestion
	"description": "Vote Delegation Certificate",
```

Reply:
@Ryun1 i have picked the description directly from https://github.com/input-output-hk/cardano-cli/issues/221

please update me on how we should name it

Reply:
@gitmachtl 

See [comment](https://github.com/input-output-hk/cardano-cli/issues/221#issuecomment-1702892317) from @CarlosLopezDeLara. 🚀

Reply:
@Ryun1 updated the file